tests/kernel/poll/src/test_poll.c

```diff
@@ -15,11 +15,16 @@ struct fifo_msg {
 
 #define SIGNAL_RESULT 0x1ee7d00d
 #define FIFO_MSG_VALUE 0xdeadbeef
+#define STACK_SIZE (1024 + CONFIG_TEST_EXTRA_STACKSIZE)
```

Comment:
I've done similar stuff in other tests to fold stacks together.  Maybe we should have some kind of ztest framework for this or something.